src/data/dropdownDataGroupedDescriptions.tsx

```diff
@@ -0,0 +1,114 @@
+import {Folder, Lock} from '~/icons';
+
+export const dropdownGroupedDataDescriptions = [
```

Comment:
Finally, let's remove data with descriptions to avoid having too many datasets to maintain. So, we will add an example with a description in another dataset. (Sorry for the last-minute changes.)